scripts/check-licenses.js

```diff
@@ -3,15 +3,19 @@
 const fs = require('fs')
 const path = require('path')
 const readline = require('readline')
-const { executeCommand, printLog, printError, logAndExit } = require('./utils')
+const { findPackages, printLog, printError, logAndExit } = require('./utils')
 
 const LICENSE_FILE = 'LICENSE-3rdparty.csv'
 
 async function main() {
-  const packageJsonPaths = await findPackageJsonPaths()
+  const packages = await findPackages()
 
-  printLog('Looking for dependencies in:\n', packageJsonPaths, '\n')
-  const declaredDependencies = withoutDuplicates(packageJsonPaths.flatMap(retrievePackageJsonDependencies)).sort()
+  printLog(
+    'Looking for dependencies in:\n',
+    packages.map((pkg) => pkg.relativePath),
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏avoid abbreviation
```suggestion
    packages.map((package) => package.relativePath),
```

Reply:
`package` is a reserved keyword it's a syntax error to use it. I usually use `pkg` in this case but I'd be fine with another alternative, like `package_`

Reply:
oh ok, got it. Nevermind then.